### DISCUSSION TITLE
How to use a GUI with stockfish?

### DISCUSSION THREAD

**Discussion by saki-osive** [official-stockfish/Stockfish#4311] [Q&A] [answered]
There are a few options mentioned in the README.md file for the GUI that can be used with the Stockfish compiler. However, is there a handy guide to leverage any of them and get started?

Thanks

**Comment by dav1312** [comment, marked as answer]
https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage#download-a-chess-gui

**Comment by pschneider1968** [comment]
If you are serious into Chess, I would recommend very much to use Scid. It's a great open source/free software chess database. I use it, in combination with Stockfish engine, to analyze and comment all my OTB games (because I still suffer from the illusion that even at my age, I could get better at chess 😁)

It has great features and is easy to use. Here are some screen shots...

![grafik](https://user-images.githubusercontent.com/36973164/209700938-0e658267-3d96-43e8-9ac8-72eb1e0006b6.png)


![grafik](https://user-images.githubusercontent.com/36973164/209701021-b9fe325f-26d8-4915-902d-219b1e751029.png)

![grafik](https://user-images.githubusercontent.com/36973164/209701100-da0e610e-5037-490b-a6ff-69d2fd5922b9.png)

![grafik](https://user-images.githubusercontent.com/36973164/209701118-3446769e-62db-4f93-898c-6ebd6843ec65.png)

![grafik](https://user-images.githubusercontent.com/36973164/209701193-42ff79da-fc55-4e24-a76c-379643d2970f.png)

![grafik](https://user-images.githubusercontent.com/36973164/209701212-4550d1f1-f1a8-405f-a887-708229c1be77.png)

**Reply by saki-osive** [reply]
Thanks @pschneider1968 for all the inputs, would look to trying this one out.
